include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,329 @@
+    //uint32_t           minSgprAllocation;
+    //uint32_t           maxSgprAllocation;
+    //uint32_t           sgprAllocationGranularity;
+    //uint32_t           vgprsPerSimd;
+    //uint32_t           minVgprAllocation;
+    //uint32_t           maxVgprAllocation;
+    //uint32_t           vgprAllocationGranularity;
+    /* ShaderCoreProperties2AMD */
+    //VkShaderCorePropertiesFlagsAMD    shaderCoreFeatures;
+    //uint32_t                          activeComputeUnitCount;
+
+    /* BlendOperationAdvancedPropertiesEXT */
+    //uint32_t           advancedBlendMaxColorAttachments;
+    //VkBool32           advancedBlendIndependentBlend;
+    //VkBool32           advancedBlendNonPremultipliedSrcColor;
+    //VkBool32           advancedBlendNonPremultipliedDstColor;
+    //VkBool32           advancedBlendCorrelatedOverlap;
+    //VkBool32           advancedBlendAllOperations;
+            
+    /* ConservativeRasterizationPropertiesEXT */
+    //float              primitiveOverestimationSize;
+    //float              maxExtraPrimitiveOverestimationSize;
+    //float              extraPrimitiveOverestimationSizeGranularity;
+    //VkBool32           primitiveUnderestimation;
+    //VkBool32           conservativePointAndLineRasterization;
+    //VkBool32           degenerateTrianglesRasterized;
+    //VkBool32           degenerateLinesRasterized;
+    //VkBool32           fullyCoveredFragmentShaderInputVariable;
+    //VkBool32           conservativeRasterizationPostDepthCoverage;
+            
+    /* CustomBorderColorPropertiesEXT */
+    //uint32_t           maxCustomBorderColorSamplers;
+
+    /* DescriptorIndexingPropertiesEXT ---> MOVED TO Vulkan 1.2 Core */
+
+    /* DiscardRectanglePropertiesEXT */
+    //uint32_t           maxDiscardRectangles;
+            
+    /* ExternalMemoryHostPropertiesEXT */
+    //VkDeviceSize       minImportedHostPointerAlignment;
+    
+    /* AccelerationStructurePropertiesKHR */
+    uint64_t           maxGeometryCount;
+    uint64_t           maxInstanceCount;
+    uint64_t           maxPrimitiveCount;
+    uint32_t           maxPerStageDescriptorAccelerationStructures;
+    uint32_t           maxPerStageDescriptorUpdateAfterBindAccelerationStructures;
+    uint32_t           maxDescriptorSetAccelerationStructures;
+    uint32_t           maxDescriptorSetUpdateAfterBindAccelerationStructures;
+    uint32_t           minAccelerationStructureScratchOffsetAlignment;
+            
+    /* RayTracingPipelinePropertiesKHR */
+    uint32_t           shaderGroupHandleSize;
+    uint32_t           maxRayRecursionDepth;
+    uint32_t           maxShaderGroupStride;
+    uint32_t           shaderGroupBaseAlignment;
+    uint32_t           shaderGroupHandleCaptureReplaySize;
+    uint32_t           maxRayDispatchInvocationCount;
+    uint32_t           shaderGroupHandleAlignment;
+    uint32_t           maxRayHitAttributeSize;
+            
+    /* Nabla */
+    uint32_t maxBufferSize;
```

Comment:
use 1.1's maxMemoryAllocationSize

Reply:
yes.

On OpenGL just do a `max` of:
- `GL_MAX_SHADER_STORAGE_BLOCK_SIZE`
- `GL_MAX_TEXTURE_BUFFER_SIZE`
- `GL_MAX_UNIFORM_BLOCK_SIZE`